llvm/unittests/Target/AIE/HazardRecognizerTest.cpp

```diff
@@ -375,3 +379,41 @@ TEST(HazardRecognizer, composeConflicting) {
     EXPECT_EQ(HR.hazard(9, C), C == -1 || C == 0);
   }
 }
+
+/// Check scoreboard conflicts from bank utilization
+TEST(HazardRecognizer, bankConflictHazard) {
+  AIE2InstrInfo InstrInfo;
+  MockHR HR(InstrInfo);
+
+  HR.emit(1, -2, /*SlotSet=*/0b0, /*MemoryBanks=*/0b1010,
```

Comment:
Nit: You emit everything with a delta of `-2`, I think we could just use `0`, which is a nicer number for reasonning :D